lib/core/test/data/Cardano/Wallet/TokenMetadata/golden1.json

```diff
@@ -72,7 +72,10 @@
         "subject": "extra fields",
         "name": { "value": "Token2" },
         "description": { "value": "description2" },
-        "acronym": { "value": "acronym2" }
+        "acronym": { "value": "acronym2" },
+        "unit": { "value": { "decimals": 14, "name": "unit2" } },
+        "url": { "value": "https://iohk.io" },
+        "logo": { "value": "QWxtb3N0IGEgbG9nbw==" }
```

Comment:
We should probably add data + checks for these new fields in the integration tests too

Reply:
Yeah. If you know precisely how, feel free :sweat_smile: